### ISSUE TITLE
Constant crashes MAC OS

### ISSUE THREAD

**Issue by Vuccappella** [ValveSoftware/Dota-2#1745]
https://gist.github.com/Vuccappella/4cc9d0a72580545ded5a - Crash log.

**Comment by Slarky** [comment]
Same problem here, I'm getting a crash every time I quit the game

"Application Specific Information:
abort() called
*** error for object 0x5ec5ed50: pointer being freed was not allocated"

https://gist.github.com/Slarky/e1808b2394a4df019f41

**Comment by mmm0nky** [comment]
Yep, I don't know how to bring up the crash log, but the data client just crashed again while in queue and now I'm banned for 30 mins. Despite the abundance of complaints on the issue in various other forums, I have not seen a single fix.

**Comment by Slarky** [comment]
It's not crashing on exit anymore here with the 2015-06-05 update.

**Comment by Slarky** [comment]
You can get crash logs via /Applications/Utilities/Console.app
